nautobot/ipam/formfields.py

```diff
@@ -44,6 +44,10 @@ class IPNetworkFormField(forms.Field):
         "invalid": "Enter a valid IPv4 or IPv6 address (with CIDR mask).",
     }
 
+    def __init__(self, allow_zero_prefix=True, *args, **kwargs):
+        self.allow_zero_prefix = allow_zero_prefix
```

Comment:
Can you please add a docstring to the `IPNetworkFormField` to document the parameter for `allow_zero_prefix`?

Reply:
Added it :)